query_expr/query_expr.cpp

```diff
@@ -8,17 +8,17 @@
 #include <graph_zeppelin_common.h>
 #include <binary_graph_stream.h>
 
-void perform_continuous_insertions(std::string binary_input, sys_config config) {
+void perform_continuous_insertions(std::string binary_input, std::string csv_out, sys_config config) {
```

Comment:
I think we should follow a similar pattern to the perform_insertions function to keep things simple.

Reply:
we can’t really do that unless we want to return a big vector of structs, since we collect more information from it than `perform_insertions`

Reply:
Hmm fair. Okay sounds good.